### PR TITLE
Add Serratus PCB

### PR DESCRIPTION
**Pull request by mechlovin**
## Description

Add Serratus TKL PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechlovin/hannah910/rev3/config.h [hunk 1]
```diff
@@ -3,4 +3,6 @@
 #define PRODUCT_ID 0x9103
 #define DEVICE_VER 0x0002
 
-#define RGBLED_NUM 28
\ No newline at end of file
+#define RGBLED_NUM 28
+
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (drashna) [inline keyboards/mechlovin/hannah910/rev3/config.h:8]: Could this change be reverted?

File: keyboards/mechlovin/infinity87/rgb_rev1/config.h [hunk 2]
```diff
@@ -23,6 +23,7 @@
 #define MATRIX_ROW_PINS { A10, A13, A14, C13, A4, A3 }
 #define MATRIX_COL_PINS { B12, B13, B14, B15, A8, A9, B11, B10, B2, B1, B0, A7, A6, A5, A2, A1, A0 }
 
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (drashna) [inline keyboards/mechlovin/infinity87/rgb_rev1/config.h:26]: Could this change be reverted?

File: keyboards/mechlovin/th1800/config.h [hunk 3]
```diff
@@ -51,3 +51,5 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define LED_CAPS_LOCK_PIN A0
 #define LED_SCROLL_LOCK_PIN A1
 #define LED_PIN_ON_STATE 0
+
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (drashna) [inline keyboards/mechlovin/th1800/config.h:55]: Could this change be reverted.

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by mechlovin** [comment]
Thank you. I have tested polling rate on my board and forgot to delete them.

**Review by drashna — APPROVED** [review]

File: keyboards/mechlovin/serratus/matrix.c [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+#ifdef DIRECT_PINS
+static pin_t direct_pins[MATRIX_ROWS][MATRIX_COLS] = DIRECT_PINS;
+#elif (DIODE_DIRECTION == ROW2COL) || (DIODE_DIRECTION == COL2ROW)
+static const pin_t row_pins[MATRIX_ROWS] = MATRIX_ROW_PINS;
+//static const pin_t col_pins[MATRIX_COLS] = MATRIX_COL_PINS;
+#endif
+
+// matrix code
+
+#ifdef DIRECT_PINS
```
Comment (fauxpark) [inline keyboards/mechlovin/serratus/matrix.c:36]: All the direct pins stuff can be removed from here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by mechlovin** [comment]
Hi. @fauxpark.
Anything else I can do?

File: keyboards/mechlovin/serratus/readme.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# serratus
+
+![serratus](https://i.imgur.com/VN38ogY.jpeg)
```
Comment (fauxpark) [inline keyboards/mechlovin/serratus/readme.md:3]:
```suggestion
![serratus](https://i.imgur.com/VN38ogYh.jpeg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**